tests/test_randprocs/test_markov_process.py

```diff
@@ -0,0 +1,12 @@
+"""Tests for Gauss-Markov processes."""
+
+from probnum import randprocs, randvars
+
+
+def test_gauss_markov_process_initrv(gauss_markov_process: randprocs.MarkovProcess):
```

Comment:
test_markov_process_initrv(markov_process: ...)

Reply:
This test explicitly tests Gauss Markov processes, so the naming is valid.